src/code_0805436C.c

```diff
@@ -20,7 +20,33 @@ extern struct_080FD5B4 gUnk_080FD5B4[];
 extern u8 gUnk_0200AF13;
 extern u8 gUnk_0200AF14;
 extern u8 gUnk_080FE1C6[];
-extern Droptable gUnk_02034398;
+/*{
+0u,
```

Comment:
What is this for?

Reply:
thats the const data for that array, but couldn't get it to match when properly extracting.